src/main/ts/configs/Standard.ts

```diff
@@ -42,33 +47,34 @@ export const base: Linter.Config = {
         format: [ 'PascalCase' ]
       }
     ],
-    '@typescript-eslint/no-duplicate-imports': 'error',
     '@typescript-eslint/no-empty-interface': 'off',
     '@typescript-eslint/no-for-in-array': 'error',
     '@typescript-eslint/no-implied-eval': 'error',
     '@typescript-eslint/no-inferrable-types': [ 'error', { ignoreParameters: true, ignoreProperties: true }],
     '@typescript-eslint/no-namespace': 'error',
-    '@typescript-eslint/no-parameter-properties': 'error',
+    '@typescript-eslint/parameter-properties': 'error',
     '@typescript-eslint/no-shadow': [ 'error', { hoist: 'all' }],
     '@typescript-eslint/no-unused-vars': [ 'warn', {
       vars: 'all',
       args: 'after-used',
+      argsIgnorePattern: '^_',
+      caughtErrors: 'all',
+      caughtErrorsIgnorePattern: '^_',
+      destructuredArrayIgnorePattern: '^_',
       ignoreRestSiblings: true,
-      argsIgnorePattern: '^_'
     }],
-    '@typescript-eslint/object-curly-spacing': [ 'error', 'always', { objectsInObjects: false }],
+    '@stylistic/object-curly-spacing': [ 'error', 'always', { objectsInObjects: false }],
     '@typescript-eslint/prefer-for-of': 'error',
     '@typescript-eslint/prefer-function-type': 'error',
-    '@typescript-eslint/quotes': [ 'error', 'single', { allowTemplateLiterals: true }],
+    '@stylistic/quotes': [ 'error', 'single', { allowTemplateLiterals: true }],
     '@typescript-eslint/require-await': 'off',
     '@typescript-eslint/restrict-plus-operands': 'off',
     '@typescript-eslint/restrict-template-expressions': 'off',
-    '@typescript-eslint/semi': [ 'error', 'always' ],
-    '@typescript-eslint/space-before-blocks': 'error',
-    '@typescript-eslint/space-before-function-paren': [ 'error', { anonymous: 'always', named: 'never' }],
-    '@typescript-eslint/space-infix-ops': 'error',
-    '@typescript-eslint/switch-exhaustiveness-check': 'error',
```

Comment:
Should we keep this rule? Is this no longer needed or handled elsewhere?

Reply:
I would also like to see this type of rule be included somehow

Reply:
Thanks, it looks like I accidentally deleted this one. To make it match the older implementation of the rule, I also need to set https://typescript-eslint.io/rules/switch-exhaustiveness-check/#considerdefaultexhaustiveforunions to `true`. 
I might look at making this rule stricter in a follow-up